core/src/main/java/io/questdb/cairo/mv/MatViewDefinition.java

```diff
@@ -193,6 +202,17 @@ private static MatViewDefinition loadMatViewDefinition(final ReadableBlock block
         }
 
         long offset = 0;
+        final int refreshType = block.getInt(offset);
+        if (refreshType != INCREMENTAL_REFRESH_TYPE) {
+            throw CairoException.critical(0)
+                    .put("unsupported refresh strategy [view=")
```

Comment:
nit: inconsistent naming strategy/type

Reply:
Fixed in 6385339fd1b1514f70520e5c8f230ef8f37eec39